src/indicators/drought/ecmwf_seasonal/processing.py

```diff
@@ -262,6 +263,54 @@ def convert_dict_to_da(da_dict):
     )
 
 
+def get_crps_ecmwf(
+    da_observations: xr.DataArray,
+    da_forecasts: xr.DataArray,
+    normalization: str = None,
+    thresh: float = None,
+) -> pd.DataFrame:
+    """
+    Assumes there is no missing data in da_observations or da_forecasts
+    :param da_observations: data-array or data-set with observed values
+    :param da_forecasts: data-array with forecasted values
+    normalization: (optional) Can be 'mean' or 'std', reanalysis metric
+    to divide the CRPS
+    param: threshold: (optional) only select values smaller or equal to
+    this number
+    :return: DataFrame with leadtime index containing the crps
+    """
+    leadtimes = da_forecasts.leadtime.values
+    df_crps = pd.DataFrame(index=leadtimes)
+    observations = da_observations.dropna(dim="time", how="all")
```

Comment:
In the docstring you say no missing data, then why do you drop NA here?

Reply:
Hehe oops I was experimenting and then cheating